packages/demo/frontend/src/components/earn/Action.tsx

```diff
@@ -61,9 +61,23 @@ export function Action({
     marketId?.address.toLowerCase() ===
       '0x4200000000000000000000000000000000000006'
 
+  // Max withdrawal for illiquid Aave market
+  const AAVE_MAX_WITHDRAW = 0.0001
+
+  // Determine display precision based on asset type
+  const isWethAsset = assetSymbol === 'WETH' || assetSymbol.includes('ETH')
```

Comment:
wait, so WETH is both ETH and WETH?

Reply:
This was needlessly broad, updated it to only `WETH`.